### PR TITLE
[UPD] base_wamas_ubl: Add new grammars BKORR, LBA, LBABQ, LBAEQ and LBAMQ

### PR DESCRIPTION
**Pull request by tuantrantg**

### CODE REVIEW

File: base_wamas_ubl/lib/wamas/const.py [hunk 1]
```diff
@@ -51,6 +56,8 @@
     "WATEKQ": "PickingResponse",
     "WEAK": "Reception",
     "WEAKQ": "ReceptionResponse",
+    "BKORR": "RandomInventoryCorrection",
```
Comment (jbaudoux) [inline base_wamas_ubl/lib/wamas/const.py]: why random?
```suggestion
    "BKORR": "InventoryCorrection",
```

File: base_wamas_ubl/lib/wamas/const.py [hunk 2]
```diff
@@ -51,6 +56,8 @@
...
+    "LBAMQ": "DailyInventoryAdjustment",
```
Comment (jbaudoux) [inline base_wamas_ubl/lib/wamas/const.py]:
```suggestion
    "LBABQ": "InventoryResponse",
    "LBAMQ": "InventoryResponse",
```

**Comment by tuantrantg** [comment]
hi @jbaudoux the PR has been updated, please review it again, thank you

**Review by jbaudoux — APPROVED** [review]

**Review by jbaudoux — APPROVED** [review]

**Pull request merged**